plugins/modules/system/launchd.py

```diff
@@ -0,0 +1,484 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2018, Martin Migasiewicz <migasiew.nk@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: launchd
+author:
+- Martin Migasiewicz (@martinm82)
+version_added: "0.1.1"
+short_description:  Manage macOS services
+description:
+- This module can be used to control launchd services on target macOS hosts.
```

Comment:
```suggestion
- Manage launchd services on target macOS hosts.
```
if the module can't be used for anything else, we can simplify the description